KomaMRIBase/src/datatypes/sequence/Grad.jl

```diff
@@ -43,6 +43,29 @@ rotz(θ::Real) = [cos(θ) -sin(θ)	0;
 				 sin(θ) cos(θ)	0;
 				 0 		0 		1]
 
+
+"""
+	R = rotation_matrix(G)
+
+Creates a rotation matrix which converts vector [0 0 1] into vector G
+"""
+rotation_matrix(G::AbstractVector{<:Real}=[0;0;0]) = begin
+# We need to create a rotation matrix which transfomrs vector [0 0 1] into vector G
+# To do this, we can use axis-angle representation, and then calculate rotation matrix with that
+# https://en.wikipedia.org/wiki/Rotation_matrix#Conversion_from_rotation_matrix_to_axis%E2%80%93angle
+REF = [0;0;1];
+# Cross product:
+global cross_prod = LinearAlgebra.cross(REF,G);
+# Rotation axis (n = axb) Normalized cross product:
+n = normalize(cross_prod);
+# Rotation angle:
+θ = asin(norm(cross_prod)/((norm(REF))*(norm(G))));
+# Rotation matrix:
+R = (norm(cross_prod)>0) ? Un(θ,n) : [1.0  0    0;
+									  0    1.0  0;
+									  0    0    1.0];							
+end
+
```

Comment:
Is this needed? `Un` does the same.

Reply:
Resolved in [55e5bd3](https://github.com/JuliaHealth/KomaMRI.jl/pull/184/commits/55e5bd3483372e37f656204c4dd3ccaa5bae9784)